src/optionals/math.h

```diff
@@ -7,6 +7,8 @@
 #include "optionals.h"
 #include "../vm/vm.h"
 
+#define FLOAT_TOLERANCE 0.00001
```

Comment:
This value was decided arbitrarily. In the future if types are added for integers we can get rid of this and make the functions only accept integers. Until then using a float tolerance seems like a good idea, as otherwise people might enter numbers that appear to be whole numbers, but due to float math imprecision from previous calculations will actually not be.